WowApiWrapper.lua

```diff
@@ -0,0 +1,82 @@
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitClass")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitExists")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitGUID")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitAffectingCombat")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetSpecialization")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetActiveSpecGroup")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetInstanceInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "IsInInstance")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetNumGroupMembers")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitIsUnit")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetRaidRosterInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetSpecialization")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetSpecializationRole")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitInVehicle")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitHasVehicleUI")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetSpellInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "SendChatMessage")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetChannelName")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitInBattleground")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitInRaid")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitInParty")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "PlaySoundFile")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "PlaySoundKitID")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetTime")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetSpellLink")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetItemInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "CreateFrame")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "IsShiftKeyDown")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetScreenWidth")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetScreenHeight")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetCursorPosition")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "random")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UpdateAddOnCPUUsage")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetFrameCPUUsage")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "debugprofilestop")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "debugstack")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "IsSpellKnown")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetTalentInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetPvpTalentInfo")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "GetAddOnEnableState")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "IsPlayerMoving")
+    WowApiWrapper:ConnectGlobalFunction(wrapper, "UnitIsDead")
+end
+
+function WowApiWrapper:ConnectGlobalFunction(wrapper, name)
+    wrapper[name] = _G[name]
```

Comment:
Connects the global function with some name to the API wrapper. This is what makes the API Wrapper just pass-through to the actual in-game function.